vNext/shared/AppInsightsCommon/src/Util.ts

```diff
@@ -288,28 +287,24 @@ export class Util {
      * Force the SDK not to store and read any data from cookies
      */
     public static disableCookies() {
-        Util._canUseCookies = false;
+        CoreUtils.disableCookies();
     }
 
     /*
      * helper method to tell if document.cookie object is available
      */
     public static canUseCookies(logger: IDiagnosticLogger): any {
-        if (Util._canUseCookies === undefined) {
-            Util._canUseCookies = false;
-
-            try {
-                Util._canUseCookies = Util.document.cookie !== undefined;
```

Comment:
this still needs to remain here. Core should be agnostic of concept of document object.

Reply:
Added back